lib/fetch/global.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+/** @type {undefined|URL} */
+let origin
```

Comment:
I would use the same logic setGlobalDispatcher uses for this.

Reply:
Is it fine to add another symbol? Wasn't sure because of the issue linked above

Reply:
Sure thing.

Reply:
done